src/ProjectTemplates/test/Templates.Tests/template-baselines.json

```diff
@@ -1689,5 +1689,93 @@
         "Program.cs"
       ]
     }
+  },
+  "blazorwasm-empty": {
+    "None": {
+      "Template": "blazorwasm-empty",
+      "Arguments": "new blazorwasm-empty",
+      "Files": [
+        "Pages/Index.razor",
+        "Properties/launchSettings.json",
+        "wwwroot/css/app.css",
+        "wwwroot/index.html",
+        "_Imports.razor",
+        "App.razor",
+        "MainLayout.razor",
+        "Program.cs"
+      ]
+    },
+    "Hosted": {
+      "Template": "blazorwasm-empty",
+      "Arguments": "new blazorwasm-empty --hosted",
+      "Files": [
+        "Client/_Imports.razor",
+        "Client/App.razor",
+        "Client/MainLayout.razor",
+        "Client/Program.cs",
+        "Client/Pages/Index.razor",
+        "Client/Properties/launchSettings.json",
+        "Client/wwwroot/index.html",
+        "Client/wwwroot/css/app.css",
+        "Server/Properties/launchSettings.json",
+        "Server/appsettings.Development.json",
+        "Server/appsettings.json",
+        "Server/Program.cs",
+        "Shared/HelloWorld.cs"
+      ]
+    },
+    "PWA": {
+      "Template": "blazorwasm-empty",
+      "Arguments": "new blazorwasm-empty --pwa",
+      "Files": [
+        "Pages/Index.razor",
+        "Properties/launchSettings.json",
+        "wwwroot/css/app.css",
+        "wwwroot/index.html",
+        "wwwroot/manifest.json",
+        "wwwroot/service-worker.js",
+        "wwwroot/service-worker.published.js",
+        "_Imports.razor",
+        "App.razor",
+        "MainLayout.razor",
+        "Program.cs"
+      ]
+    },
+    "NoHttps": {
+      "Template": "blazorwasm-empty",
+      "Arguments": "new blazorwasm-empty --no-https",
+      "Files": [
+        "Pages/Index.razor",
+        "Properties/launchSettings.json",
+        "wwwroot/css/app.css",
+        "wwwroot/index.html",
+        "_Imports.razor",
+        "App.razor",
+        "MainLayout.razor",
+        "Program.cs"
+      ]
+    },
+    "PWAHosted": {
+      "Template": "blazorwasm-empty",
+      "Arguments": "new blazorwasm-empty --pwa --hosted",
+      "Files": [
+        "Client/_Imports.razor",
+        "Client/App.razor",
+        "Client/MainLayout.razor",
+        "Client/Program.cs",
+        "Client/Pages/Index.razor",
+        "Client/Properties/launchSettings.json",
+        "Client/wwwroot/index.html",
+        "Client/wwwroot/manifest.json",
+        "Client/wwwroot/service-worker.js",
+        "Client/wwwroot/service-worker.published.js",
+        "Client/wwwroot/css/app.css",
+        "Server/Properties/launchSettings.json",
+        "Server/appsettings.Development.json",
+        "Server/appsettings.json",
+        "Server/Program.cs",
+        "Shared/HelloWorld.cs"
```

Comment:
Hmm I wonder if just an empty `HelloWorld` without additional context may be a bit confusing. Maybe we should consider renaming this to something like `SharedClass.cs` or `SharedModel.cs`, and then having a helpful comment in the actual file? Would love to hear other's opinions on this as well 😄

Reply:
I can rename the .cs file and add a short description. I can also potentially change this to a .razor file (since the default template holds .razor files in its `Shared` folder), and add a comment like "This .razor file can hold a shared component". Let me know which option you prefer.

Reply:
I think a `.cs` file may be more appropriate here.